### ISSUE TITLE
Fatal error on webdav access during maintenance mode

### ISSUE THREAD

**Issue by MichaIng** [nextcloud/server#6824]
```
Fatal | webdav | Sabre\DAV\Exception\ServiceUnavailable: System in maintenance mode.[internal function] OCA\DAV\Connector\Sabre\MaintenancePlugin->checkMaintenanceMode(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))/var/www/nextcloud/3rdparty/sabre/event/lib/EventEmitterTrait.php - line 105: call_user_func_array(Array, Array)/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 466: Sabre\Event\EventEmitter->emit('beforeMethod', Array)/var/www/nextcloud/3rdparty/sabre/dav/lib/DAV/Server.php - line 254: Sabre\DAV\Server->invokeMethod(Object(Sabre\HTTP\Request), Object(Sabre\HTTP\Response))/var/www/nextcloud/apps/dav/lib/Server.php - line 258: Sabre\DAV\Server->exec()/var/www/nextcloud/apps/dav/appinfo/v2/remote.php - line 33: OCA\DAV\Server->exec()/var/www/nextcloud/remote.php - line 162: require_once('/var/www/nextcl...'){main}
```

On every webdav access try during maintenance mode I get a fatal error. Of course every access during maintenance mode will totally fail, but as this is totally expected an wanted I don't understand why this fact needs to spam up my Nextcloud log that prominent with fatal errors, the highest error class, as if something horrible happened.

Can't all maintenance related errors be reduced to [info], [debug] or completely dropped from Nextcloud log? I mean everyone knows and expects this, by enabling maintenance mode and nobody will able to see this errors as reminder (at least not in web ui) while maintenance mode is active...

Sorry for exaggerated language, but lately I got quite a lot annoyed by this 😄.

_If this is really NOT expected behaviour and might be a bug related to my system/client, of course I will fill every information I can. But for now I guess this is not a bug._

**Comment by nickvergessen** [comment]
This has been fixed for nextcloud 13 already:
https://github.com/nextcloud/server/pull/6132

**Comment by MichaIng** [comment]
Works like a charm, thanks very much to you guys!

**Comment by MorrisJobke** [comment]
Also fixed in 12.0.4.
